web-registry/src/components/molecules/ProjectPageMetadata/ProjectPageMetadata.tsx

```diff
@@ -38,7 +37,6 @@ const ProjectPageMetadata: React.FC<React.PropsWithChildren<MetadataProps>> = ({
   // CFC
   const cfcProjectId = metadata?.['regen:cfcProjectId'];
   const projectDesignDocument = metadata?.['regen:projectDesignDocument'];
-  const offsetProtocol = metadata?.['regen:offsetProtocol'];
```

Comment:
I believe we might want to do the same for Toucan approved methodologies below

Reply:
I think we do yes, done here: [515aeb7](https://github.com/regen-network/regen-web/pull/1937/commits/515aeb73f7e68f65815073ff929eed903759b50e)